backend/src/routes/users/users.query.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import axios, { AxiosResponse } from 'axios';
+
+/**
+ * Attaches a given access token to an MS Graph API call
+ * @param {string} endpoint - REST API endpoint to call
+ * @param {string} accessToken - Raw access token string
+ * @returns {Promise<any>} - Result of the request
+ */
+async function user_connection(endpoint: string, accessToken: string): Promise<any> {
+    const options = {
+        headers: {
+            Authorization: `Bearer ${accessToken}`,
+        },
+    };
+    console.log(`Requête effectuée à ${endpoint} à : ` + new Date().toString());
```

Comment:
Should be in english. Moreover, that's not necessary, you're using a logger above